docs/catalog/cloud/instal-supported-add-ons-and-extensions.md

```diff
@@ -0,0 +1,34 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- kubernetes
+- role:cloud-engineer
+- size:medium
+---
+
+# Install supported add-ons and extensions
+
+## Description
+Add-ons and extensions covered by the editor support policy provide additional and supported functionality, while allowing you to benefit from the latest performance improvements and energy optimizations throughout your service lifecycle.
```

Comment:
I am not clear about this pattern. Can we discuss this in the next call?
Few comments - Pls provide context for editor support policy